src/js/components/DataTable/DataTable.js

```diff
@@ -219,62 +223,35 @@ const DataTable = ({
     console.warn('DataTable cannot combine "size" and "resizeble".');
   }
 
+  const [page, showItem] = normalizeShow(showProp, 'dataTable');
+  const [setPage, currentItems, currentPage] = usePagination({
+    data: adjustedData,
+    paginationProps: { page, showItem, step, ...paginate },
```

Comment:
Since we're already in "usePagination" context, can we simplify this down to `{data: adjustedData, show, step, ...paginate}`?

Reply:
Simplified in recent updates.